src/components/main/RecruitFloat.tsx

```diff
@@ -1,116 +1,91 @@
-        ref={floatDetailRef}
+        className="fixed bottom-0 z-[200] h-16 w-full max-w-[1600px]"
+        onMouseEnter={showDetail}
+        onMouseLeave={closeDetail}
       >
-        <div className="pb-2 border-b-white border-b-2 flex justify-between px-4">
-          <div className="flex gap-4 items-baseline">
-            <div className="uppercase text-3xl">
-              {RECRUIT_FLOAT.ECONO_GENERTAION_RECRUIT_EN}
-            </div>
-            <div>{RECRUIT_FLOAT.ECONO_GENERTAION_RECRUIT_KR}</div>
-          </div>
-          <div className="flex gap-8">
-            <div className="flex gap-4">
-              <div className="flex flex-col items-center mr-4">
-                <div className="text-3xl">{days}</div>
-                <div className="text-[0.5rem] uppercase">
-                  {RECRUIT_FLOAT.DAY}
-                </div>
-              </div>
-              <div className="flex flex-col items-center">
-                <div className="text-3xl opacity-90">{hours}</div>
-                <div className="text-[0.5rem] uppercase">
-                  {RECRUIT_FLOAT.HOUR}
-                </div>
-              </div>
-              <Image
-                className="mb-4"
-                src={require("/public/icons/colon.svg").default}
-                alt="colon"
-              />
-              <div className="flex flex-col items-center">
-                <div className="text-3xl opacity-90">{minutes}</div>
-                <div className="text-[0.5rem] uppercase">
-                  {RECRUIT_FLOAT.MINUTE}
-                </div>
+        <div
+          className="group absolute -bottom-4 rounded-t-3xl bg-black px-8 pb-8 pt-4 text-2xl font-bold uppercase text-white"
+          ref={floatRef}
+        >
+          <HoverText status={recruitStatus} days={days} />
+        </div>
+        <div
+          className="absolute -bottom-[200%] w-[calc(100%-6rem)] rounded-t-3xl bg-black px-8 pb-8 pt-4 font-semibold text-white"
+          ref={floatDetailRef}
+        >
+          <div className="flex justify-between border-b-2 border-b-white px-4 pb-2">
+            <div className="flex items-baseline gap-4">
+              <div className="text-3xl uppercase">
+                {RECRUIT_FLOAT.ECONO_GENERTAION_RECRUIT_EN}
               </div>
-              <Image
-                className="mb-4"
-                src={require("/public/icons/colon.svg").default}
-                alt="colon"
-              />
-              <div className="flex flex-col items-center">
-                <div className="text-3xl opacity-90">{seconds}</div>
-                <div className="text-[0.5rem] uppercase">
-                  {RECRUIT_FLOAT.SECOND}
-                </div>
+              <div>
+                {RECRUIT_FLOAT.ECONO_GENERTAION_RECRUIT_KR}{" "}
+                {getRecruitJosaByRecruitStatus(recruitStatus)}
               </div>
```

Comment:
저의 과거의 유산이라고 생각하고...는 어떠신가요??
```suggestion
              <div>
                `${RECRUIT_FLOAT.ECONO_GENERTAION_RECRUIT_KR} ${getRecruitJosaByRecruitStatus(recruitStatus)}`
               </div>
```

Reply:
@loopy-lim ㅋㅋㅋㅋ 좋습니다
지금 열띤 토론중이라 해당 부분 변경되었는데, 이런 스타일로 구성해볼께요.

Reply:
https://econovation-2018.slack.com/archives/C07HX1GTR6C/p1724946397273599

Reply:
보이려나? ㅋㅋ

Reply:
ㅋㅋㅋㅋㅋ ㅠㅠㅠㅠㅠ 안보이는 것으로 ㅠㅠ

Reply:
@loopy-lim 

<img width="563" alt="image" src="https://github.com/user-attachments/assets/49b2a9c5-3d55-4b05-bced-5eda5877659c">

이렇게 변동 되었습니다

[51a7b35](https://github.com/JNU-econovation/econo-homepage/pull/127/commits/51a7b3537ab073ac36f864ccaeb717be08a44b7d):

Reply:
앗!! 달러표시가 빠졋어요!!

Reply:
ㅋㅋㅋ nbsp 오래간만에 보네요 ㅋㅋㅋㅋ 좋아용 ㅋㅋㅋ

Reply:
ㅋㅋㅋㅋ 최선이였슈